### PR TITLE
Add the gamescript

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

File: controllers/game.php [hunk 1]
```diff
@@ -14,7 +14,14 @@ public function create() {
                 go();
             }
 
-            go( 'game', 'update', [ 'gameid' => $game->id ] );
+            switch ( $this->outputFormat ) {
+                case 'text':
+                    return $game->id;
```
Comment (dionyziz) [inline controllers/game.php]: Return value indicates success; you should print the game id to stdout.

File: gamescript.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/zsh
+php run game create;
+php run game update gameid=$? finishit=yes;
```
Comment (dionyziz) [inline gamescript.sh]: functional test this

File: gamescript.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/zsh
```
Comment (dionyziz) [inline gamescript.sh]: prefer bash for scripting

**Comment by pkakelas** [comment]
LGTM

**Comment by dionyziz** [comment]
fix build

File: tests/gamescript.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            exec( "ENVIRONMENT=test ./gamescript.sh", $output );
+            $created = true;
+            try {
+                $lastGame = Game::getLastGame();
+            }
+            catch ( ModelNotFoundException $e ) {
+                $created = false;
+            }
+            $this->assertTrue( $created, 'A game must be created' );
+            $this->assertTrue( $lastGame->ended, 'The game must be finished' );
```
Comment (dionyziz) [inline tests/gamescript.php:28]: This test is inadequate, because there are no users; therefore the game will be finished regardless of whether gamescript.sh actually calls for it to be finished. Recommend adding a few bots to the game artificially in this functional test to ensure the game finishes after a non-zero amount of rounds. An additional assertion (using `assert` to indicate expectations, not part of the unit test) to make sure the number of rounds was non-zero is appropriate.

File: helpers/file.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                    recurse_copy( $src . '/' . $file, $dst . '/' . $file );
+                }
+                else {
+                    copy( $src . '/' . $file, $dst . '/' . $file );
+                }
+            }
+        }
+        closedir( $dir );
+    }
+    function recurse_delete( $src ) {
```
Comment (dionyziz) [inline helpers/file.php]: test

File: helpers/file.php [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<?php
+    function recurse_copy( $src, $dst ) {
```
Comment (dionyziz) [inline helpers/file.php]: test

File: tests/gamescript.php [hunk 7]
```diff
@@ -0,0 +1,32 @@
+<?php
+    require_once 'helpers/file.php';
+    class GamescriptTest extends FunctionalTest {
+        protected $mockPath = 'bots/mock';
```
Comment (dionyziz) [inline tests/gamescript.php:4]: Move fixtures to tests/ directory

**Comment by dionyziz** [comment]
Fix build
